README.md

```diff
@@ -1,4 +1,7 @@
 # ⛩ node-gateway
+[![Test](https://github.com/satsuma-xyz/node-gateway/actions/workflows/test.yml/badge.svg)](https://github.com/satsuma-xyz/node-gateway/actions/workflows/test.yml)
```

Comment:
Think you may need to rebase off main against since I force pushed to amend a commit msg